examples/provider/terraform.tfvars

```diff
@@ -1,3 +1,3 @@
 username = "admin"
-password = "xxxxxxxxx"
-validate_certs = true
+password = "Netapp1!"
```

Comment:
Please revert it back to what it was.

Reply:
I've reverted it back.
I had specifically staged `examples/data-sources/netapp-ontap_security_certificate` & `examples/data-sources/netapp-ontap_security_certificates`; I'm wondering how this file got modified!